notebooks/README.md

```diff
@@ -0,0 +1,13 @@
+# Useful Jupyter Notebooks to get started with Anomalib
+
+## 0. Training and Inference
+
+| Notebook        | GitHub                                                               | Colab                                                                                                                                                                                                                    |
+| --------------- | -------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Getting Started | [001-getting-started](001-getting-started/001-getting-started.ipynb) | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openvinotoolkit/anomalib/blob/development/notebooks/001-getting-started/001-getting-started.ipynb) |
+
+## 1. Benchmarking and Hyperparameter Optimization
+
+| Notebook     | GitHub                                                      | Colab                                                                                                                                                                                                              |
+| ------------ | ----------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Benchmarking | [101-benchmarking](101-benchmarking/101-benchmarking.ipynb) | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openvinotoolkit/anomalib/blob/development/notebooks/101-benchmarking/101-benchmarking.ipynb) |
```

Comment:
The benchmarking button won't work as it links to a blob in development. As we don't have it yet on that branch, it will not work. It works when you replace development with the name of this branch.